src/pages/voting/[votingSlug].tsx

```diff
@@ -179,6 +184,25 @@ const VotingDetail = () => {
     }
   })
 
+  let stateText = votingProposalDetails?.state
+  if (votingProposalDetails?.state === VotingStatus.CLOSED) {
+    const highestVotedIndex = votingProposalDetails?.scores.indexOf(
+      Math.max(...votingProposalDetails?.scores)
+    )
+    if (
+      votingProposalDetails?.choices[highestVotedIndex] === VotingChoices.FOR
+    ) {
+      stateText = VotingOutcomes.PASSED
+    } else if (
+      votingProposalDetails?.choices[highestVotedIndex] ===
+      VotingChoices.AGAINST
+    ) {
+      stateText = VotingOutcomes.FAILED
+    } else {
+      stateText = VotingOutcomes.ABSTAINED
+    }
+  }
```

Comment:
just the same as the other comment from `VotingCard.tsx`

```suggestion
  const [stateText, setStateText] = useState(votingProposalDetails?.state)
  
  useEffect(() => {
    if (votingProposalDetails?.state === VotingStatus.CLOSED) {
      const highestVotedIndex = votingProposalDetails?.scores.indexOf(Math.max(...votingProposalDetails?.scores))
      if (votingProposalDetails.choices[highestVotedIndex] === VotingChoices.FOR) {
        setStateText(VotingOutcomes.PASSED)
      } else if (votingProposalDetails.choices[highestVotedIndex] === VotingChoices.AGAINST) {
        setStateText(VotingOutcomes.FAILED)
      } else {
        setStateText(VotingOutcomes.ABSTAINED)
      }
    }
  
  }, [state])
```